apollo-client/src/main/java/com/ctrip/framework/apollo/spring/util/BeanRegistrationUtil.java

```diff
@@ -29,6 +29,15 @@
  * @author Jason Song(song_s@ctrip.com)
  */
 public class BeanRegistrationUtil {
+  private static final Map<String, String> BD_MAPPING = new ConcurrentHashMap<>();
+
+  static {
+    BD_MAPPING.put(
```

Comment:
This doesn't seem like an elegant solution, but it just works as BeanRegistrationUtil is used in limited scenarios.